### PR TITLE
Adds Cyborg Android Types

### PR DESCRIPTION
**Pull request by ChromeFoxxity**
## About the Pull Request

Makes the cyborgs a little more lively, and realistic-looking, depending on what real life AI technology is going for.

## Why It's Good For The Game

To make the cyborg players a little more cool looking.

## Changelog

:cl:
add: Normal Forensics module. Robot ballistics.
imageadd: Added several android icons.
imagedel: Replaced/Removed a few useless icons.
/:cl:

### CODE REVIEW

**Comment by checkraisefold** [comment]
thats not a borg thats an ipc!!!
however I am fine with this since it fits with the aesthetic more and looks much better anyway

**Comment by checkraisefold** [comment]
Re-merge dev to fix tests

File: code/modules/mob/living/silicon/robot/modules/module_security.dm [hunk 1]
```diff
@@ -37,19 +34,19 @@
...
+		"Android - Guard" = "securityrobot",
+		"Android - NCO Guard" = "securityncorobot"
 	)
 	equipment = list(
 		/obj/item/device/flash,
 		/obj/item/borg/sight/hud/sec,
 		/obj/item/handcuffs/cyborg,
 		/obj/item/melee/baton/robot,
-		/obj/item/gun/energy/gun/secure/mounted,
+		/obj/item/gun/energy/gun/mounted/mk9/mounted/rubber,
```
Comment (projectkepler-ru) [inline code/modules/mob/living/silicon/robot/modules/module_security.dm]: this is a horrible idea

File: code/modules/projectiles/guns/energy/secure.dm [hunk 2]
```diff
@@ -73,6 +73,58 @@
...
+	icon = 'icons/obj/gun.dmi'
+	icon_state = "MK9"
+	modifystate = null
+	projectile_type = /obj/item/projectile/bullet/pistol/rubber
+	fire_delay = 2
+	self_recharge = 1
+	use_external_power = 0
+	has_safety = FALSE
+
+/obj/item/gun/energy/gun/mounted/p90/mounted
```
Comment (projectkepler-ru) [inline code/modules/projectiles/guns/energy/secure.dm]: the two subtype below are not being referenced anywhere

**Review by projectkepler-ru — COMMENTED** [review]
ADD AN IMAGE

File: code/modules/mob/living/silicon/robot/modules/module_security.dm (hunk 1 above)
Comment (checkraisefold) [reply code/modules/mob/living/silicon/robot/modules/module_security.dm]: agreed this is not good

Comment (ChromeFoxxity) [reply code/modules/mob/living/silicon/robot/modules/module_security.dm]: It is an unobtainable module that cannot be played unless its a "Crisis", which I'm not really sure when that counts.

**Pull request merged**